packages/liveblocks-react/src/factory.tsx

```diff
@@ -175,19 +175,19 @@ export function createRoomContext<
       client.enter(roomId, {
         initialPresence: frozen.initialPresence,
         initialStorage: frozen.initialStorage,
-        unstable_batchedUpdates: frozen.unstable_batchedUpdates,
         shouldInitiallyConnect: frozen.shouldInitiallyConnect,
-      } as RoomInitializers<TPresence, TStorage>)
+        unstable_batchedUpdates: frozen.unstable_batchedUpdates,
+      })
     );
 
     React.useEffect(() => {
       setRoom(
         client.enter(roomId, {
           initialPresence: frozen.initialPresence,
           initialStorage: frozen.initialStorage,
+          shouldInitiallyConnect: frozen.shouldInitiallyConnect,
           unstable_batchedUpdates: frozen.unstable_batchedUpdates,
-          withoutConnecting: frozen.shouldInitiallyConnect,
-        } as RoomInitializers<TPresence, TStorage>)
+        })
```

Comment:
Ignore the change to the type annotation and the whitespace here. The core of the fix here is the change of the key in this object: `withoutConnecting` (old name) → `shouldInitiallyConnect` (new name).

Reply:
✅